ingest-app/docs/api.md

```diff
@@ -226,11 +226,14 @@ Requests could fail for several reasons when communicating with the CMR as descr
 
 Ingest validation errors can take one of two shapes. General error messages will be returned as a list of error messages like the following:
```

Comment:
I think `Ingest validation errors can take one of two forms.` should be above the General Error section.

Reply:
done.